feat(changelog): redistm-changed-default-username-is-now 2023-08-21 (#2054)
scaleway/docs-content: changelog/august2023/2023-08-21-redistm-changed-default-username-is-now.mdx

---

> title: Default username is now supported by Managed database for Redis

Reviewer: ```suggestion
title: Managed database for Redis now supports the default username
```

status: changed
author:

>     fullname: 'Join the #[database] channel on Slack.'

Reviewer 2: ```suggestion
    fullname: 'Join the #database channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-08-21
category: databases
product: redistm
---

> You can now use the `default` username on your managed database for Redis. This enables some software integrations where specific usernames are not supported. Anonymous connection remains unavailable.

Reviewer 2: ```suggestion
You can now use the `default` username on your managed database for Redis. This enables some software integrations where specific usernames are not supported. Anonymous connections remain unavailable.
```

> You can now use the `default` username on your managed database for Redis. This enables some software integrations where specific usernames are not supported. Please note the anonymous connection remains unavailable.

Reviewer: ```suggestion
You can now use the `default` username on your managed database for Redis. This enables some software integrations where specific usernames are not supported. Anonymous connection remains unavailable.
```
